sensing/autoware_pointcloud_preprocessor/schema/ring_outlier_filter_node.schema.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Ring Outlier Filter Node",
+  "type": "object",
+  "definitions": {
+    "ring_outlier_filter": {
+      "type": "object",
+      "properties": {
+        "distance_ratio": {
+          "type": "number",
+          "description": "distance_ratio",
+          "default": "1.03",
+          "minimum": 0
+        },
+        "object_length_threshold": {
+          "type": "number",
+          "description": "object_length_threshold",
+          "default": "0.1",
+          "minimum": 0
+        },
+        "num_points_threshold": {
+          "type": "integer",
+          "description": "num_points_threshold",
+          "default": "4",
+          "minimum": 0
+        },
+        "max_rings_num": {
+          "type": "integer",
+          "description": "max_rings_num",
+          "default": "128",
+          "minimum": 1
+        },
+        "max_points_num_per_ring": {
+          "type": "integer",
+          "description": "Set this value large enough such that HFoV / resolution < max_points_num_per_ring",
+          "default": "4000",
+          "minimum": 0
+        },
+        "publish_outlier_pointcloud": {
+          "type": "boolean",
+          "description": "Flag to publish outlier pointcloud and visibility score. Due to performance concerns, please set to false during experiments.",
+          "default": "false"
+        },
+        "min_azimuth_deg": {
+          "type": "number",
+          "description": "The left limit of azimuth for visibility score calculation",
+          "default": "0.0",
+          "minimum": 0
+        },
+        "max_azimuth_deg": {
```

Comment:
https://github.com/vividf/autoware.universe/blob/137b54689675871ce011871e6e50549ec791f9a3/sensing/autoware_pointcloud_preprocessor/src/outlier_filter/ring_outlier_filter_node.cpp#L395

Given how these parameters are used it would be worth it to add a maximum value (e,g, 720 degrees would not work)

Reply:
Thanks for the check! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8468/commits/cd0be8e437d3e99c92e615a2569f06fb44408584